### PR TITLE
upgrade spark312 to spark313

### PR DESCRIPTION
**Pull request by Le-Zheng**
## Description
Spark_core_2.12 requires spark 3.1.3 to avoid a high Vulnerability. We  need to upgrade spark 312 to 313
org.apache.spark:spark-core_2.12@3.1.2 Upgrade to 3.1.3
image

### CODE REVIEW

File: python/dev/release_default_mac_spark313.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #
 
 # This is the default script with maven parameters to release all the bigdl sub-packages
-# built on top of Spark 3.1.2 for mac.
+# built on top of Spark 3.1.3 for mac.
```
Comment (hkvision) [inline python/dev/release_default_mac_spark313.sh:20]: Line 30-31 needs to be modified as well.

**Comment by hkvision** [comment]
@leonardozcm Take a look if there's anywhere missing

**Comment by leonardozcm** [comment]
It will take an effort to adapt jenkins scripts from `bash python/dev/release_default_linux_spark**312**.sh` to `bash python/dev/release_default_linux_spark**313**.sh`, what about keeping the name `release_default_linux_spark**312**.sh` for now and let's talk about name modification later.

**Comment by hkvision** [comment]
> It will take an effort to adapt jenkins scripts from `bash python/dev/release_default_linux_spark**312**.sh` to `bash python/dev/release_default_linux_spark**313**.sh`, what about keeping the name `release_default_linux_spark**312**.sh` for now and let's talk about name modification later.

Then what about directly changing to `release_default_linux_spark2/3` in the next step?

**Comment by hkvision** [comment]
@pinggao187 Check the Jenkins jobs?

**Comment by leonardozcm** [comment]
> Then what about directly changing to `release_default_linux_spark2/3` in the next step?

I agree

**Comment by Le-Zheng** [comment]
Ok, in this PR, just add release_default_linux_spark313.sh and release_default_mac_spark313.sh. No changes for other scripts.
We need to replace the scripts on Jenkins/github action with new spark313 script.
Let's enable spark313 first, and then remove spark312 or modify name to default spark3?

**Comment by leonardozcm** [comment]
> Ok, in this PR, just add release_default_linux_spark313.sh and release_default_mac_spark313.sh. No changes for other scripts. We need to replace the scripts on Jenkins/github action with new spark313 script. Let's enable spark313 first, and then remove spark312 or modify name to default spark3?

sure

**Comment by Le-Zheng** [comment]
Ok, will merge and test.

**Pull request merged**